Resources/Prototypes/_EE/Entities/Mobs/Player/humanoid.yml

```
@@ -0,0 +1,24 @@
+    - type: GrantImplant
+      implants: [ MindShieldImplant ]
+    - type: GhostRole
+      name: ghost-role-information-ert-security-name
+      description: ghost-role-information-ert-security-description
+      rules: ghost-role-information-nonantagonist-rules
+      raffle:
+        settings: short
+      job: ERTSecurity
+    - type: Loadout
+      prototypes: [ ERTSecurityGearEVALaser ]
+      roleLoadout: [ RoleSurvivalEVA ]
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)